.gitignore

```diff
@@ -191,3 +191,12 @@ cython_debug/
 # Dependency directories (remove the comment below to include it)
 # vendor/
 
+# iDRAC credentials
+idrac_vars.yaml
```

Comment:
can you add a `idrac_vars.example.yaml` file?

Reply:
ok